tensorflow/core/kernels/boosted_trees/boosted_trees.proto

```diff
@@ -162,9 +162,7 @@ message TreeEnsemble {
 }
 
 // DebugOutput contains outputs useful for debugging/model interpretation, at
-// the individual example-level. Debug outputs that are available to the user
-// are: 1) Directional feature contributions (DFCs) 2) Node IDs for ensemble
-// prediction path 3) Leaf node IDs.
+// the individual example-level.
```

Comment:
i would suggest to keep this description 
 // Debug outputs that are available to the user
// are: 1) Directional feature contributions (DFCs) 2) Node IDs for ensemble
// prediction path 3) Leaf node IDs. 
And add that you also now have leaf ids available